src/meltano/core/utils/__init__.py

```diff
@@ -54,107 +56,115 @@ def wrapper(*args, **kwargs):  # noqa: WPS430
 
 
 # from https://github.com/jonathanj/compose/blob/master/compose.py
-def compose(*fs):
-    """
-    Create a function composition.
-    :type *fs: ``iterable`` of 1-argument ``callable``s
-    :param *fs: Iterable of 1-argument functions to compose, functions will be
-        applied from last to first, in other words ``compose(f, g)(x) ==
-        f(g(x))``.
-    :return: I{callable} taking 1 argument.
+def compose(*fs: Callable[[Any], Any]):
+    """Create a composition of unary functions.
+
+    Parameters:
+        fs: Unary functions to compose.
+
+    Examples:
+        ```python
+        compose(f, g)(x) == f(g(x))
+        ```
+
+    Return:
+        The composition of the provided unary functions, which itself is a unary function.
     """
     return functools.reduce(lambda f, g: lambda x: f(g(x)), compact(fs), lambda x: x)
 
 
 # from http://www.dolphmathews.com/2012/09/slugify-string-in-python.html
 def slugify(s):
-    """
-    Simplifies ugly strings into something URL-friendly.
+    """Normalize strings into something URL-friendly.
+
     >>> slugify("[Some] _ Article's Title--")
     'some-articles-title'
     """
-
-    # "[Some] _ Article's Title--"
-    # "[some] _ article's title--"
+    # "[Some] _ Article's Title--" -> "[some] _ article's title--"
```

Comment:
These changes were made because otherwise some `flake8` plugin thinks that we have commented out code. This is better than just `noqa`-ing all of these comments.